PerformanceTesting/ecl/02eb_smallsorts2.ecl

```diff
@@ -15,23 +14,127 @@
+//xversion algo='parquicksort',numrows=6000
+//xversion algo='parquicksort',numrows=10000
+//xversion algo='parquicksort',numrows=12000
+//xversion algo='parquicksort',numrows=16000
+//xversion algo='parquicksort',numrows=24000
+//xversion algo='parquicksort',numrows=32000
+//xversion algo='parquicksort',numrows=48000,nothor
+//xversion algo='parquicksort',numrows=64000,nothor
+//xversion algo='parquicksort',numrows=96000,nothor
+//xversion algo='parquicksort',numrows=128000,nothor
+//xversion algo='parquicksort',numrows=192000,nothor
+//xversion algo='parquicksort',numrows=256000,nothor
+//xversion algo='parquicksort',numrows=384000,nothor
+//xversion algo='parquicksort',numrows=512000,nothor
+
+//xversion algo='mergesort',numrows=1000
+//xversion algo='mergesort',numrows=2000
+//xversion algo='mergesort',numrows=4000
+//xversion algo='mergesort',numrows=6000
+//xversion algo='mergesort',numrows=10000
+//xversion algo='mergesort',numrows=12000
+//xversion algo='mergesort',numrows=16000
+//xversion algo='mergesort',numrows=24000
+//xversion algo='mergesort',numrows=32000
+//xversion algo='mergesort',numrows=48000,nothor
+//xversion algo='mergesort',numrows=64000,nothor
+//xversion algo='mergesort',numrows=96000,nothor
+//xversion algo='mergesort',numrows=128000,nothor
+//xversion algo='mergesort',numrows=192000,nothor
+//xversion algo='mergesort',numrows=256000,nothor
+//xversion algo='mergesort',numrows=384000,nothor
+//xversion algo='mergesort',numrows=512000,nothor
+
+//xversion algo='heapsort',numrows=1000
+//xversion algo='heapsort',numrows=2000
+//xversion algo='heapsort',numrows=4000
+//xversion algo='heapsort',numrows=6000
+//xversion algo='heapsort',numrows=10000
+//xversion algo='heapsort',numrows=12000
+//xversion algo='heapsort',numrows=16000
+//xversion algo='heapsort',numrows=24000
+//xversion algo='heapsort',numrows=32000
+//xversion algo='heapsort',numrows=48000,nothor
+//xversion algo='heapsort',numrows=64000,nothor
+//xversion algo='heapsort',numrows=96000,nothor
+//xversion algo='heapsort',numrows=128000,nothor
+//xversion algo='heapsort',numrows=192000,nothor
+//xversion algo='heapsort',numrows=256000,nothor
+//xversion algo='heapsort',numrows=384000,nothor
+//xversion algo='heapsort',numrows=512000,nothor
+
 import ^ as root;
-algo := #IFDEFINED(root.algo, 'quicksort');
-numRows := #IFDEFINED(root.numrows, 100);
- 
+algo := #IFDEFINED(root.algo, 'parmergesort');
+numRows := #IFDEFINED(root.numrows, 512000);
+numIters := #IFDEFINED(root.numIters, 10000/5);
+
 idRecord := RECORD
     UNSIGNED id;
 END;
 
 import Std.System.Debug;
 
-idRecords := DATASET(10000, TRANSFORM(idRecord, SELF.id := COUNTER-1));
+idRecords := DATASET(numIters, TRANSFORM(idRecord, SELF.id := COUNTER-1));
 
 idRecord t(idRecord l) := TRANSFORM
     ids := DATASET(numRows + (l.id * NOFOLD(0)), TRANSFORM(idRecord, SELF.id := HASH64(COUNTER)));
     s := SORT(ids, id, STABLE(algo));
     check := SORTED(NOFOLD(s), id, ASSERT);
```

Comment:
Is this 'check' needed if we not use it later? (What is the meaning of the line 136? I can't find that ASSERT parameter in our docs.)